quickstarts/azure/azure-app-configuration/config.yml

```diff
@@ -0,0 +1,41 @@
+slug: azure-app-configuration
+description: |-
+ ## What is Azure App Configuration?
+
+  Azure App Configuration provides a service to centrally manage application settings and feature flags.
+
+  ### New Relic Azure AppConfiguration quickstart features
+
+  A standard dashboard that tracks key indicators like daily storage usage, http incoming request count, http incoming request duration and throttled http request count. It runs custom queries and visualizes the data immediately.
+
+  ### Why monitor Azure AppConfiguration with New Relic?
+
+  [New Relic Azure App Configuration](https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-app-configuration-monitoring-integration/) monitoring quickstart empowers you to track the performance of Azure App Configuration via different metrics including daily storage usage, http incoming request count, http incoming request duration and throttled http request count.
+  
+   Our integration features a standard dashboard that provides interactive visualizations to explore your data, understand context, and get valuable insights.
+  
+  Start ingesting your Azure data today and get immediate access to our visualization dashboards so you can optimize your Azure service. 
+summary: |-
+  Monitoring Azure App Configuration is critical to track the performance through key metrics. Download New Relic Azure App Configuration monitoring quickstart to get a pre-built dashboard tailored to monitor your Azure App Configuration Service.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+  - New Relic Partner
+title: Azure App Configuration
+documentation:
+  - name: Azure App Configuration installation docs.
+    description: |
+      Monitor Azure App Configuration by connecting Azure to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-app-configuration-monitoring-integration/
+keywords:
+  - azure
+  - app configuration
+  - product partnerships
+installPlans:
```

Comment:
We have deprecated installPlans. Please remove any instance of installPlan within your configs